tensorrt_llm/executor/result.py

```diff
@@ -76,6 +91,8 @@ def __post_init__(self):
             self.token_ids = []
         if self.logprobs is None:
             self.logprobs = []
+        if self.prompt_logprobs is None:
+            self.prompt_logprobs = []
```

Comment:
For setting default values with containers, we can declare the parameters in dataclass as
```python
logprobs: LogprobResults = field(default_factory=list)
```
This eliminates the need for the code here.

So are other parameters here.